org.bluej.BlueJ.json

```diff
@@ -1,22 +1,20 @@
 {
   "app-id" : "org.bluej.BlueJ",
   "branch" : "stable",
-  "runtime" : "org.gnome.Platform",
-  "runtime-version" : "40",
-  "sdk" : "org.gnome.Sdk",
-  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk11"],
   "command": "org.bluej.BlueJ",
   "finish-args" : [
     "--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
     "--share=network",
-    "--socket=x11",
-    "--share=ipc",
+    "--socket=fallback-x11",
     "--socket=wayland",
+    "--share=ipc",
     "--device=dri",
     "--socket=pulseaudio",
-    "--filesystem=host",
-    "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
-    "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
There is something that needs to be done to migrate dconf, not just removing this.

See:
https://docs.flatpak.org/en/latest/sandbox-permissions.html#dconf-access